rollup.config.js

```diff
@@ -0,0 +1,34 @@
+import copy from 'rollup-plugin-copy';
+import del from 'rollup-plugin-delete';
+import dynamicImportVars from '@rollup/plugin-dynamic-import-vars';
+import glob from 'glob-all';
+import resolve from '@rollup/plugin-node-resolve';
+
+const jsGlob = [
+	'@(components|controllers|directives|helpers|mixins|templates)/**/*.js',
+	'!**/*@(test|axe|visual-diff).js',
+];
+const nonJsGlob = [
+	'@(components|controllers|directives|helpers|mixins|templates)/**/*.*',
+	'*.*',
+	'!**/*.@(js|md|json)',
+	'!**/screenshots/**/*',
+];
+
+export default {
+	input: glob.sync(jsGlob),
+	output: { dir: 'build', format: 'es', preserveModules: true },
+	external: ['puppeteer', '@brightspace-ui/visual-diff', '@open-wc/testing', 'sinon'],
```

Comment:
By marking these as external, rollup just leaves them alone - which means that if we tried to actually use a file with one of these imports, it would break.

Technically, only `'@brightspace-ui/visual-diff'` now needs to be marked external - it's used in `tooltip-helper.js`, `dialog-helper.js`, and `dropdown-helper.js`. None of those helpers are imported by anything that'll display in a webpage - they're only used as helpers for `visual-diff.js` files.

The other externals here were from before I excluded `.test.js` etc. But it probably doesn't hurt to keep them in.